doc/CONTRIBUTING.md

```diff
@@ -9,6 +9,71 @@ release except releases labeled as "stable". Stable releases can be found at
 [https://qgreenland.org/explore](https://qgreenland.org/explore)!
 
 
+## Storage
+
+QGreenland storage locations are currently in the `data/` subdirectory of this
+repository. This can be overridden by creating `data/working-storage` and
+`data/private-archive` symlinks to your preferred storage locations.
+
+We plan to support storage location overrides using environment variables in
+the future.
+
+
+### Working storage
+
+```
+data/working-storage
+```
+
+All file outputs of QGreenland are written to working storage. Must be
+read-write.
+
+
+#### Dataset fetch storage
+
+```
+data/working-storage/fetch-datasets
+```
+
+Datasets are fetched for their original locations and written to this
+directory.
+
+
+#### WIP storage
+
+```
+data/working-storage/wip-layers
+data/working-storage/wip-package
+```
+
+All intermediate steps are written to these directories.
+
+
+#### Release storage
+
+```
+data/working-storage/release-layers
+data/working-storage/release-packages
+```
+
+Finalized layers and projects are stored in these directories.
+
+
+### "Private archive" storage
+
+```
+data/private-archive
+```
+
+While we prefer to only include publicly-archived data, we do have some
```

Comment:
`While we` -> `While this project prefers`?

Reply:
1b8e9b1